github/repos.go

```diff
@@ -1037,3 +1037,38 @@ func (s *RepositoriesService) ReplaceAllTopics(ctx context.Context, owner, repo
 
 	return t.Names, resp, nil
 }
+
+// TransferRequest represents a request to transfer a repository.
+type TransferRequest struct {
+	NewOwner string `json:"new_owner,omitempty"`
```

Comment:
According to GitHub API docs, `new_owner` is required, so I don't think `omitempty` is appropriate here.

Reply:
Excellent catch. Fixed.